tests/queries_tests/template_task_test.js

```diff
@@ -36,12 +48,15 @@ describe('templateTask', () => {
     expect(templateTask).to.be.a('object')
   )
 
-  it('should return all template tasks ordered by days_to_complete', () =>
-    templateTask.getAll().then( templateTasks => {
-      expect( templateTasks[0].title ).to.equal('Breath')
-      expect( templateTasks[1].title ).to.equal('Massage')
-    })
-  )
+  templateTask.getAll().then( templateTasks => {
+
+  const taskTitles = templateTask.map( task => task.title )
+    it('should return all template tasks ordered by days_to_complete and title', () =>
+    expect(taskTitles[0]).to.equal('Breath'),
```

Comment:
this can be changed to
expect(taskTitles).to.equal(['Breath', 'Lunch', 'Massage'])